signin_parser_spec.rb

```diff
@@ -12,26 +12,30 @@
   end
 
   it "should be signed in and sign out" do
-    signed = @signin_parse.sign_in(@url, @login, @password, @enter)
+    signed = @signin_parse.sign_in(@url, @login, @password, @enter, @quit)
```

Comment:
Since you developing parser for a specific site, i don't think you need to to have url and quit button as params at all, you can make them as constants in class for ex.